scripts/rsp_to_gcm_test.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
- Does it run with Python 2 & 3?
- Better make this script pass `pycodestyle`/`pep8`

Reply:
Yes it seems to work with both Python 2 and 3.
I'll fix the style stuff.